src/corn-model.ts

```diff
@@ -124,7 +133,20 @@ export function initCornModel(simulationElt: HTMLElement | null, params?: IModel
   if (simulationElt) {
     simulationElt.appendChild(interactive.getEnvironmentPane());
   }
-
+  env.addRule(new Rule({
+    test(agent: Agent) {
+      return env.date > 50 && env.date < 60 && agentIsEgg(agent) && agent.get('hatched') === false;
```

Comment:
From my reading of the code, `env.date` doesn't take year/seasons into account, so once this becomes a multi-year simulation this will need to be something like `env.date % yearLength`.